src/devenv.nix

```diff
@@ -3,6 +3,14 @@ let
   examples = ../examples;
   lib = pkgs.lib;
   version = lib.fileContents ./modules/latest-version;
+  inherit (pkgs)
+    coreutils
+    findutils
+    gnugrep
+    jq
+    docopts
+    util-linuxMinimal;
+  devenv-yaml = import ./devenv-yaml.nix { inherit pkgs; };
 in
 pkgs.writeScriptBin "devenv" ''
   #!/usr/bin/env bash
```

Comment:
do we want to reference here also our bash 🤔

Reply:
Oof, good point :+1: I've changed to it use `pkgs.bash`!